components/common-go/cgroups/cgroup.go

```diff
@@ -5,108 +5,21 @@
 package cgroups
 
 import (
-	"os"
-	"path/filepath"
-	"strings"
+	"github.com/containerd/cgroups"
+	v2 "github.com/containerd/cgroups/v2"
 )
 
-const DefaultCGroupMount = "/sys/fs/cgroup"
-
-type CgroupSetup int
-
-const (
-	Unknown CgroupSetup = iota
-	Legacy
-	Unified
-)
-
-func (s CgroupSetup) String() string {
-	return [...]string{"Legacy", "Unified"}[s]
-}
-
-func GetCgroupSetup() (CgroupSetup, error) {
-	controllers := filepath.Join(DefaultCGroupMount, "cgroup.controllers")
-	_, err := os.Stat(controllers)
-
-	if os.IsNotExist(err) {
-		return Legacy, nil
-	}
-
-	if err == nil {
-		return Unified, nil
-	}
-
-	return Unknown, err
-}
-
 func IsUnifiedCgroupSetup() (bool, error) {
-	setup, err := GetCgroupSetup()
-	if err != nil {
-		return false, err
-	}
-
-	return setup == Unified, nil
-}
-
-func IsLegacyCgroupSetup() (bool, error) {
-	setup, err := GetCgroupSetup()
-	if err != nil {
-		return false, err
-	}
-
-	return setup == Legacy, nil
+	return cgroups.Mode() == cgroups.Unified, nil
```

Comment:
We have discussed using containerd or runc cgroup libs, and we decided not to depend on any libs. WDYT?
https://github.com/gitpod-io/gitpod/pull/8471#discussion_r819542056

Reply:
We already have that library as a dependency. Not sure that argument is valid?